cli/compose/loader/full-example.yml

```diff
@@ -280,6 +280,15 @@ volumes:
       foo: "bar"
       baz: 1
 
+  another-volume:
+    name: "user_specified_name"
+    driver: vsphere
+
+    driver_opts:
+      # Values can be strings or numbers
+      foo: "bar"
+      baz: 1
+
```

Comment:
Also (not sure if that needs to be done in this file, or another one); can we have a test that tests if setting both `volume.name` and `volume.external.name` produces the "conflicting options" error? i.e.;

```yaml
  external-volume4-fail:
    # name cannot be provided both as volume.name and volume.external.name
    name: custom-name
    external:
      name: conflicting-name
```